client/src/components/Page/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { withStyles, createStyles } from '@material-ui/core/styles';
+import /*type*/ { WithStyles, Theme } from '@material-ui/core/styles';
```

Comment:
What's up with this `/*type*/`?

Reply:
So there's this new syntax that CRA doesn't have available yet. [link](https://stackoverflow.com/questions/59833120/parsing-error-expected-in-import-type)

When the syntax becomes available, I want to use it

Essentially I like to split `type` imports  from other imports and have them closer to the top of the import list 
Makes TS more readable I guess

Reply:
Ooo ok cool